renderer/test/file_system_access_browsertest.cc

```diff
@@ -49,14 +62,30 @@ class FileSystemAccessBrowserTest : public InProcessBrowserTest {
 
  protected:
   net::EmbeddedTestServer https_server_;
+  base::test::ScopedFeatureList scoped_feature_list_;
 };
 
-IN_PROC_BROWSER_TEST_F(FileSystemAccessBrowserTest, FilePicker) {
+IN_PROC_BROWSER_TEST_P(FileSystemAccessBrowserTest, FilePicker) {
+  EXPECT_EQ(IsFileSystemAccessAPIEnabled(),
+      base::FeatureList::IsEnabled(blink::features::kFileSystemAccessAPI));
+
   const GURL url = https_server_.GetURL("/simple.html");
   ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), url));
 
-  auto result = content::EvalJs(main_frame(), "self.showOpenFilePicker()");
-  EXPECT_TRUE(result.error.find("self.showOpenFilePicker is not a function") !=
-              std::string::npos)
-      << result.error;
+  if (IsFileSystemAccessAPIEnabled()) {
+    auto result =
+        content::EvalJs(main_frame(), "typeof self.showOpenFilePicker");
+    EXPECT_TRUE(*result.value.GetIfString() == "function")
```

Comment:
the dereference will crash if `GetIfString` returns `nullptr`, so we won't see `DebugString()` value at all. I'd go with:
```
ASSERT_TRUE(result.value.is_string());
EXPECT_EQ(result.value.GetString(), "function");
```
or just:
```
EXPECT_EQ(result.value.GetString(), "function");
```

Reply:
Great catch. Fixed in 8aa596d with the second option.